src/services/docGenService.ts

```diff
@@ -92,13 +89,11 @@ export class DocGenService {
       >["0"];
 
       try {
-        const result = enableNewDocsPanel
-          ? await this.altimateRequest.generateModelDocsV2({
-              ...baseRequest,
-              user_instructions: message.user_instructions,
-              follow_up_instructions: message.follow_up_instructions,
-            })
-          : await this.altimateRequest.generateModelDocs(baseRequest);
```

Comment:
apis also can be removed, if not getting used anywhere.

Reply:
done